src/rpc/server.cpp

```diff
@@ -470,6 +470,163 @@ UniValue CRPCTable::execute(const JSONRPCRequest &request) const
+            "getdifficulty",
+            "getindexinfo",
+            "getmemoryinfo",
+            "getmempoolancestors",
+            "getmempooldescendants",
+            "getmempoolentry",
+            "getmempoolinfo",
+            "getmininginfo",
+            "getnettotals",
+            "getnetworkhashps",
+            "getnetworkinfo",
+            "getnewaddress",
+            "getnodeaddresses",
+            "getpeerinfo",
+            "getrawchangeaddress",
+            "getrawmempool",
+            "getrawtransaction",
+            "getreceivedbyaddress",
+            "getreceivedbylabel",
+            "getrpcinfo",
+            "gettransaction",
+            "gettxout",
+            "gettxoutproof",
+            "gettxoutsetinfo",
+            "getunconfirmedbalance",
+            "getwalletinfo",
+            "getzmqnotifications",
+            "help",
+            "importaddress",
+            "importdescriptors",
+            "importprunedfunds",
+            "importpubkey",
+            "importwallet",
+            "invalidateblock",
+            "joinpsbts",
+            "keypoolrefill",
+            "listaddressgroupings",
+            "listbanned",
+            "listlabels",
+            "listlockunspent",
+            "listreceivedbyaddress",
+            "listreceivedbylabel",
+            "listsinceblock",
+            "listtransactions",
+            "listunspent",
+            "listwalletdir",
+            "listwallets",
+            "loadwallet",
+            "lockunspent",
+            "logging",
+            "mockscheduler",
+            "ping",
+            "preciousblock",
+            "prioritisetransaction",
+            "pruneblockchain",
+            "psbtbumpfee",
+            "reconsiderblock",
+            "removeprunedfunds",
+            "rescanblockchain",
+            "savemempool",
+            "scantxoutset",
+            "sendmany",
+            "sendrawtransaction",
+            "sendtoaddress",
+            "setban",
+            "setlabel",
+            "setmocktime",
+            "setnetworkactive",
+            "settxfee",
+            "setwalletflag",
+            "stop",
+            "submitblock",
+            "submitheader",
+            "testmempoolaccept",
+            "unloadwallet",
+            "upgradewallet",
+            "uptime",
+            "utxoupdatepsbt",
+            "validateaddress",
+            "verifychain",
+            "verifymessage",
+            "verifytxoutproof",
+            "waitforblock",
+            "waitforblockheight",
+            "waitfornewblock",
+            "walletcreatefundedpsbt",
+            "walletlock",
+            "walletprocesspsbt",
+        };
+        std::string params;
+        if (logging_allow_list.find(request.strMethod) == logging_allow_list.end()) {
+            params = "****";
+        } else {
+            for (size_t i = 0; i < request.params.size(); ++i) {
+                if (i > 0) {
+                    params += ",";
```

Comment:
Not sure here, perhaps separate the params with `", "` (comma + space)

now
```
2021-08-19T11:45:46Z rpc=getmempoolancestors(b6a5ed05bc71c8ccc5316,true)
```
comma+space
```
2021-08-19T11:45:46Z rpc=getmempoolancestors(b6a5ed05bc71c8ccc5316, true)
```